pygmt/__init__.py

```diff
@@ -1,11 +1,17 @@
 # pylint: disable=missing-docstring
 #
-# The main API for PyGMT.
+# PyGMT is a library for processing geospatial and geophysical data and making
+# publication quality maps and figures. It provides a Pythonic interface for
+# the Generic Mapping Tools (GMT), a command-line program widely used in the
+# Earth Sciences.
 #
-# All of PyGMT is operated on a "modern mode session" (new to GMT6). When you
-# import the pygmt library, a new session will be started automatically. The
-# session will be closed when the current Python process terminates. Thus, the
-# Python API does not expose the `gmt begin` and `gmt end` commands.
+# Main Features
```

Comment:
So who are the new PyGMT users in the future? those who already use GMT or who know little about GMT? For the first kind of users, we may want to emphasize what the advantage of PyGMT as compared to using GMT in Bash or other Shell (more pythonic). For the second one, we may list more GMT functions (https://docs.generic-mapping-tools.org/latest/modules.html has a good module categories which we may refer to)

Reply:
That's a good point to think about. As you mention @core-man we should try to reach both, new users and advanced "classic" GMT users. On the other side, I think we should not overload the descripiton and only add the most important things.

Reply:
Yes, I agree that we should only keep the most important things. [Project goals](https://github.com/GenericMappingTools/pygmt#project-goals) in README.rst is a good summary for the advantage of PyGMT as compared to classic command-line GMT, and maybe 1~2 points are enough. As your revisions, we should focus on GMT's main features which are also discussed in the last comment.

Reply:
Added some points.